### DISCUSSION TITLE
来点瑟图~

### DISCUSSION THREAD

**Discussion by shadio9958** [share121/inter-knot#2734] [其他]
鼠鼠搓手.jpg
![GVOJm1jXsAAogVj](https://github.com/user-attachments/assets/f1bb3d54-f736-4454-975c-9f2f4963fc7b)

**Comment by MocoTEL** [comment]
我去这个cos的好

**Comment by MrDucker** [comment]
没想到就连简这样身材极好的角色也能还原！

**Comment by HKFDBHZN** [comment]
![image](https://github.com/user-attachments/assets/4b7732be-bf3e-4f64-b534-979741ef8d66)

**Reply by HKFDBHZN** [reply]
(bushi

**Comment by shouyeren7** [comment]
我去，太好看了

**Comment by nina-mmk** [comment]
好涩

**Comment by yzy028** [comment]
好看

**Comment by 111snow-cherry** [comment]
豪堪！

**Comment by Manbow1997** [comment]
图来源于推特：@Saruei_
分享涩图是好的，但我还是要批评你了：你来源也不标明，图片也下载不了，让我们干看着，这种行为很不好，也让我好找。这种行为做派要改正！严正申明！！！
原图在下（386KB）
![Squeak 🐭 #JaneDoe #zzzero](https://github.com/user-attachments/assets/40fd5273-e499-48d2-81a8-604314e2b9bd)

**Reply by shadio9958** [reply]
忘了

**Reply by shawnho1029** [reply]
去推看了，他不會還是個V吧?

**Comment by LuAn-afl** [comment]
太喜欢了<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >

**Comment by Timekeeper2006** [comment]
好还原

**Comment by ZZZ2098** [comment]
豪堪
